staging/storm/README.md

```diff
@@ -22,15 +22,15 @@ Source is freely available at:
 This example assumes you have a Kubernetes cluster installed and
 running, and that you have installed the ```kubectl``` command line
 tool somewhere in your path. Please see the [getting
-started](../../docs/getting-started-guides/) for installation
+started](https://kubernetes.io/docs/setup/) for installation
```

Comment:
[getting started guides]

Reply:
The new submission has been modified to a more consistent links